ppafm/logging_utils.py

```diff
@@ -0,0 +1,181 @@
+import logging
+import os
+import sys
+from typing import Any, Optional, Union
+
+# Setup the root logger for the ppafm package. All other loggers will be derived from this one.
+_root_logger = logging.getLogger("ppafm")
+_log_format = "%(message)s"
+_log_handler = None
+_log_path = None
+
+# Setup another logger for performance benchmarking
+_perf_logger = logging.getLogger("ppafm.perf")
+_perf_logger.propagate = False
```

Comment:
I do not understand here - why `propagate = False`

Reply:
It's so that the messages don't get doubled by the root logger.